src/guides/v2.4/graphql/mutations/add-wishlist-items-to-cart.md

```diff
@@ -0,0 +1,155 @@
+
+## Syntax
+
+```graphql
+mutation {
+  addWishlistItemsToCart (
+    wishlistId: ID!
+    wishlistItemIds: [ID!]
+  ) {
+    AddWishlistItemsToCartOutput
+  }
+}
+```
+
+## Example usage
+
+The following example moves two items from a wishlist to the cart.
+
+**Request:**
+
+```graphql
+mutation {
+  addWishlistItemsToCart(
+    wishlistId: 1
+    wishlistItemIds: [2, 3])
+  {
+    status
+    add_wishlist_items_to_cart_user_errors {
+      code
+      message
+    }
+    wishlist {
+      id
+      items_v2 {
+        items {
+          id
+          product {
+            uid
+            sku
+            name
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+**Response:**
+
+``` json
+{
+  "data": {
+    "addWishlistItemsToCart": {
+      "status": true,
+      "add_wishlist_items_to_cart_user_errors": [],
+      "wishlist": {
+        "id": "1",
+        "items_v2": {
+          "items": [
+            {
+              "id": "1",
+              "product": {
+                "uid": "MTM=",
+                "sku": "24-WB07",
+                "name": "Overnight Duffle"
+              }
+            },
+            {
+              "id": "4",
+              "product": {
+                "uid": "MTEwMA==",
+                "sku": "WH04",
+                "name": "Miko Pullover Hoodie"
+              }
+            },
+            {
+              "id": "5",
+              "product": {
+                "uid": "MTIyOA==",
+                "sku": "WJ01",
+                "name": "Stellar Solar Jacket"
+              }
+            },
+            {
+              "id": "6",
+              "product": {
+                "uid": "MTcyNA==",
+                "sku": "WT03",
+                "name": "Nora Practice Tank"
+              }
+            },
+            {
+              "id": "7",
+              "product": {
+                "uid": "MTY5Mg==",
+                "sku": "WT01",
+                "name": "Bella Tank"
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+## Input attributes
+
+The `addWishlistItemsToCart` mutation requires the following input.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`wishlistItemIds`| [ID!] | An array of IDs presenting products to be added to the cart. If no IDs are specified, all items in the wish list will be added to the cart
+`wishlistId`| ID! | The unique ID of the wish list
+
+## Output attributes
+
+The `addWishlistItemsToCart` mutation returns the status of the operation as well an array of any errors that occurred.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`add_wishlist_items_to_cart_user_errors` | [[WishlistCartUserInputError!](#WishlistCartUserInputError)] | Indicates why the attempt to add items to the wish list was not successful
+`status` | Boolean! | Indicates whether the attempt to add items to the cart was successful
+`wishlist` | [Wishlist!](#Wishlist) | Contains the wish list with all items that were successfully added
```

Comment:
I wonder if the description should be something like " Contains items in the wishlist after adding to cart" . But it is up to you @keharper if we need to align the description with other mutations where "wishlist" is being returned in the response or if we need to update description depending on the context